<launch>
<node pkg="rrt_explore" type="rrt_explore" respawn="false" name="rrt_explore" output="screen">
  <param name="robot_base_frame" value="base_footprint"/>
  <param name="costmap_topic" value="map"/>
  <param name="costmap_updates_topic" value="map_updates"/>
  <param name="visualize" value="true"/>
  <param name="planner_frequency" value="0.2"/>
  <param name="progress_timeout" value="60.0"/>
  <param name="potential_scale" value="3.0"/>
  <param name="orientation_scale" value="0.0"/>
  <param name="gain_scale" value="3.0"/>
  <param name="transform_tolerance" value="0.5"/>
  <param name="min_frontier_size" value="0.75"/>

  <param name="early_stop_enable" value="true"/>
  <param name="steer_distance" value="20"/>
  <param name="rrt_max_iter" value="5000"/>
</node>
</launch>
